K3poControlTests/K3poControlTest.m

```diff
@@ -0,0 +1,103 @@
+/**
```

Comment:
Do we still need this? It seems like it was added just to test the K3PO control protocol implementation against the K3PO server. Now all tests seem to be commented out.